README.md

```diff
@@ -108,10 +103,6 @@ Install this project with:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- **Add task**
-- **Remove task**
 - **Hover over tasks**
 
```

Comment:
- [ ] [OPTIONAL]
Kindly endeavor to remove the guide note below the MIT license, on _line 144_ of the ReadMe.